jira/client.py

```diff
@@ -196,7 +197,7 @@ class JIRA(object):
     AGILE_BASE_URL = GreenHopperResource.AGILE_BASE_URL
 
     def __init__(self, server=None, options=None, basic_auth=None, oauth=None, jwt=None, kerberos=False,
-                 validate=False, get_server_info=True, async=False, logging=True, max_retries=3, proxies=None):
+                 validate=False, get_server_info=True, async=False, max_retries=3, proxies=None):
         """Construct a JIRA client instance.
```

Comment:
This will break the old code that using this parameter. We need to refactor this so the old cold will not break, probably via `**kwargs` but one thing that I don't like about `**kwargs` is that they make much harder to read the documentation / API.